ethvm/main.cpp

```diff
@@ -274,6 +283,8 @@ int main(int argc, char** argv)
 	state.addBalance(sender, value);
 
 	unique_ptr<SealEngineFace> se(ChainParams(genesisInfo(networkName)).createSealEngine());
+	LastBlockHashes lastBlockHashes;
+	EnvInfo const envInfo(blockHeader, lastBlockHashes, 0);
```

Comment:
Looks like previously ethvm didn't work for the code containing BLOCKHASH. Now any BLOCKHASH will return 0